doc/source/train/user-guides/checkpoints.rst

```diff
@@ -105,8 +105,8 @@ Here are a few examples of saving checkpoints with different training frameworks
             :start-after: __lightning_save_example_start__
             :end-before: __lightning_save_example_end__
 
-        You can always get the saved checkpoint path from :attr:`result.checkpoint <ray.train.Result.checkpoint>` and
-        :attr:`result.best_checkpoints <ray.train.Result.best_checkpoints>`.
+        You can always get the saved checkpoint path from :attr:`result.checkpoint <ray.train.Result>` and
+        :attr:`result.best_checkpoints <ray.train.Result>`.
```

Comment:
All these result APIs we'd want to still point to the V1 API?

Reply:
No, should be V2. There were some doc build issues that I couldn't fix so I just worked around them. (not able to find certain attributes of Result).